libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -129,20 +129,24 @@ export const NavbarIconButtonWrapper = styled('div')`
   align-items: center;
   width: ${({theme}) => theme.spacing(6.5)};
   height: ${({theme}) => theme.spacing(6.5)};
-  color: ${({theme}) => theme.palette.common.white};
+  color: white;
 
   ${({theme}) => theme.breakpoints.up('md')} {
+    width: ${({theme}) => theme.spacing(7.5)};
     height: ${({theme}) => theme.spacing(7.5)};
 
     svg {
+      width: ${({theme}) => theme.spacing(4.5)};
       height: ${({theme}) => theme.spacing(4.5)};
     }
   }
 
   ${({theme}) => theme.breakpoints.up('lg')} {
+    width: ${({theme}) => theme.spacing(12.5)};
     height: ${({theme}) => theme.spacing(12.5)};
 
     svg {
+      width: ${({theme}) => theme.spacing(6.5)};
       height: ${({theme}) => theme.spacing(6.5)};
```

Comment:
dont think we need those reverts

Reply:
yeah I didn't notice the color, it should be ${({theme}) => theme.palette.common.white};

but for width, for me the hamburger is squeezed 
![Uploading Screenshot 2024-03-20 at 12.36.45.png…]()

Reply:
Ahhh! This should probably be font-size then and no need to set width/height. As the width & height is set to 1em. Good catch!